### ISSUE TITLE
Can the project reproduce CIFAR 100 results?

### ISSUE THREAD

**Issue by wandering007** [kevinzakka/DenseNet#1]
Hi, I tried to use [the official densenet pytorch version](https://github.com/gpleiss/efficient_densenet_pytorch/blob/master/models/densenet.py) but cannot reproduce the results reported on CIFAR 100. The accuracy I got is only ~65% and the loss/error did not significantly drop (even no change) when  the learning rate decreases (i.e. at epoch 150 and 225). I was using pytorch v0.2.

Also I test densenet torch version, It works as the paper claims. I don't know what is wrong with pytorch version... Need some suggestions. Thanks.
